autouri/gcsuri.py

```diff
@@ -301,8 +317,13 @@ def get_blob(self, new=False) -> Blob:
     def get_bucket_path(self) -> Tuple[str, str]:
         """Returns a tuple of URI's S3 bucket and path.
         """
-        bucket, path = self.uri_wo_scheme.split(GCSURI.get_path_sep(), 1)
-        return bucket, path        
+        arr = self.uri_wo_scheme.split(GCSURI.get_path_sep(), 1)
```

Comment:
Would be better to be more explicit about the second argument.

```suggestion
        arr = self.uri_wo_scheme.split(GCSURI.get_path_sep(), maxsplit=1)
```

Reply:
Fixed as you suggested.